APIs/SweccAPI.py

```diff
@@ -60,4 +64,31 @@ async def reset_password(self, discord_username, id):
 
         response = requests.post(f"{self.url}/members/reset-password/", headers=self.headers, json=data)
         data = response.json()
-        return f"https://interview.swecc.org/#/password-reset-confirm/{data['uid']}/{data['token']}/"
\ No newline at end of file
+        return f"https://interview.swecc.org/#/password-reset-confirm/{data['uid']}/{data['token']}/"
```

Comment:
Why are we returning this particular value here (the password reset confirm thing)?

Reply:
You're probably not seeing the rest of the `reset_password` function. Expand diff on the left and it'll make sense

Reply:
if not though lmk

Reply:
Oh you're right, I see it now